hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/delete/batch2/DeleteExpungeSvcImpl.java

```diff
@@ -48,19 +50,26 @@ public DeleteExpungeSvcImpl(
 	}
 
 	@Override
-	public int deleteExpunge(List<JpaPid> theJpaPids, boolean theCascade, Integer theCascadeMaxRounds) {
+	public <PT extends IModelJson> int deleteExpunge(
+			List<JpaPid> theJpaPids, boolean theCascade, Integer theCascadeMaxRounds, PT theWorkChunk) {
```

Comment:
question: Why bring the whole WorkChunk type into this?  You only need the chunk id (String) and that would avoid this whole generic thing.

Suggestion: If you do want the object instead of just the id, why use the generic here?  WorkChunk is a public api type, and is shared with Mongo.  This will work if you delete the generics bit and change the type of theWorkChunk to WorkChunk
```suggestion
	public int deleteExpunge(
			List<JpaPid> theJpaPids, boolean theCascade, Integer theCascadeMaxRounds, WorkChunk theWorkChunk) {
```

Reply:
See comment above about bringing WorkChunk into this. 

I could bring WorkChunk into here by adding the generic type into `IDeleteExpungeSvc` interface.